### ISSUE TITLE
Add some examples of using the parameters in the geocoder section of docs

### ISSUE THREAD

**Issue by souperneon** [mapzen/mapzen.js#217]
It will helpful to see the usage of the different customizations available. Leaflet Geocoder docs has this and so does Turn-by-Turn.

**Comment by rmglennon** [comment]
So more like this stuff @souperneon and things from the SoTM-US workshop?

https://github.com/mapzen/leaflet-geocoder#examples and everything under https://github.com/mapzen/leaflet-geocoder/tree/master/examples?

There was this weird time when I'd heard that some of the shortcut methods were being deprecated in favor of `params`...and no one knows where that idea came from...so time to add on more examples.

**Comment by souperneon** [comment]
Yes please! Linking to that is a good start but the syntax is different. First time users may not know how to adapt the syntax to Mapzen.js

**Comment by rmglennon** [comment]
Ha, oh yeah, I was going to rewrite them and integrate here! ;) Wanted to make sure this kind of content was what you were suggesting.

**Comment by souperneon** [comment]
👍
